test/plugin/test_out_elasticsearch.rb

```diff
@@ -298,16 +298,25 @@ def test_configure
     }
     instance = driver(config).instance
 
-    assert_equal nil, instance.client.transport.options[:transport_options][:headers]["Content-Encoding"]
+    if Gem::Version.create(::Elasticsearch::Transport::VERSION) >= Gem::Version.new("7.14.0")
+      assert_equal nil, instance.client.transport.transport.options[:transport_options][:headers]["Content-Encoding"]
```

Comment:
Hmm, are there any good predicate to express this condition?

```ruby
if elasticsearch_transport_layer_decoupling?
...
end
```

Reply:
Fixed in 5b3c83028190522638af8b21f91542904b307613